CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributing to React Announcer
+
+The following sections describe the guidelines for contributing to Think Company's React Announcer project.
+
+## Submitting Issues
+
+We would be happy to hear from you about how we can make the React Announcer even better. Please follow these guidelines when submitting issues.
+
+* Search the issues log and verify that the same issue hasn't already been reported
+* Use a clear and descriptive title for the issue
+* Describe the exact steps which reproduce a problem, or a thorough description of a suggested enhancement, including as many details as possible
+* Describe the current behavior, and explain the behavior you expected to see instead, and why
+* Include screenshots and resulting code whenever possible
+
+## Contributing Code
+
+All code should be submitted via Pull Requests, following the procedure below.
+
+1. Fork, then clone the repo
+
+    git clone git@github.com:thinkcompany/react-a11y-announcer.git
+
+2. Push to your fork and submit a pull request 
+
+	https://github.com/thinkcompany/react-a11y-announcer/pulls
+
+Before creating a pull request, please ensure that an issue exists for the corresponding change. If an issue does not exist, please create a new one using the guidelines in "Submitting Issues" above.
+
+Things that will increase the chances that your pull request is accepted:
+
+* Include tests for your code
+* Follow the Think Company [development standards](https://bitbucket.org/thinkcompany/think_dev-standards/src/72ba3233dff0accd9781e200cafcc599352da1cd/development-standards/?at=master)
```

Comment:
Let's use the evergreen URL for this: http://standards.dev.thinkcompany.com/ - in case this changes down the road